content/intro-to-storybook/ember/en/composite-component.md

```diff
@@ -22,9 +22,7 @@ A composite component isn’t much different than the basic components it contai
 
 Start with a rough implementation of the `TaskList`. You’ll need to import the `Task` component from earlier and pass in the attributes and actions as inputs.
 
-```handlebars
-{{!-- app/components/task-list.hbs--}}
-
+```hbs:title=app/components/task-list.hbs
```

Comment:
It doesn't seem like we have syntax highlighting for handlebars. You can load it up here: https://github.com/chromaui/learnstorybook.com/blob/master/src/styles/formatting.js#L5 

```js
require(`prismjs/components/prism-handlebars`);
```

Reply:
ok, this might be a nice to have since we don't have syntax highlighting for `.vue` and `.svelte` files either.

Reply:
including the prism-handlebars component as you mentioned is breaking the app. I imported it and updated the references accordingly and i get this when i fire up the site and open `http://localhost:8000/intro-to-storybook`
![devmode with prism handlebars](https://user-images.githubusercontent.com/22988955/109573323-c3bde180-7ae6-11eb-9c72-1ac54065183d.png)
the dev tools
<img width="1440" alt="devtools with prism handlebars" src="https://user-images.githubusercontent.com/22988955/109573335-c91b2c00-7ae6-11eb-8b2d-917a0f634d2c.png">

It seems that the prism handlebars is broken. I thought that it could be the case of some caching issue and purged the cache and ran it again and the same outcome. Would like to include syntax highlighting for Ember, but it seems that not now until it's fixed

Reply:
ok, no worries then